website/pages/[slug].js

```diff
@@ -0,0 +1,52 @@
+import Layout from 'components/Layout';
+import { serialize } from 'next-mdx-remote/serialize';
+import { MDXRemote } from 'next-mdx-remote';
+import { getFileBySlug, getFiles, getNavbarContent } from 'lib/mdx';
+import { H1, H2, Text, Subtitle } from '@strapi/design-system/Text';
```

Comment:
Can you use the [Typography component](https://design-system-git-ds-website-strapijs.vercel.app/?path=/story/design-system-technical-components-typography--variants)